README.md

```diff
@@ -1,4 +1,29 @@
-# otm5-change-requests
-Tracking and reporting bugs and change requests of the OTM5 specification.
+# Welcome to the OTM5 change request tracker
 
-TODO: this page will _soon_ contain a description of the process followed to make changes to the [OTM5 specification](https://otm5.opentripmodel.org/). Stay tuned..
+If you have ideas for the [OTM5 specification][otm5-spec] or found a bug, please report them here.
```

Comment:
```suggestion
If you have ideas for the [OTM5 specification][otm5-spec] or found a bug (in either the rendering or the specification itself), please report them here.
```

Reply:
If it's ok for you, I'll add a separate paragraph in each section explaining what a bug or change request is. That makes it more explicit imho.